File: cpp/example/src/RmElement/rmElement.cpp
#include "rmElement.h"

int rmElement::naive(vector<int>& nums,int val){
    int ptid=0;
    for (int i=0;i<nums.size();i++){
        if (nums[i]!=val){
            int tmp = nums[ptid];
            nums[ptid]=nums[i];
            nums[i] = tmp;
            ptid+=1;
        }
    }

    return ptid;
}

File: py/solns/rmElement/rmElement.py

class Solution:
    @staticmethod
    def naive(nums,val):
        ptid = 0 
        for i in range(len(nums)):
            if nums[i]!=val:
                nums[ptid],nums[i]=nums[i],nums[ptid]
                ptid+=1
        return ptid
